lib/tasks/graphql.rake

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+namespace :graphql do
+  desc 'Outputs the current schema'
+  task write: :environment do
+    generated = GraphQL::Schema::Printer.print_schema(OntohubBackendSchema)
+    File.open(Rails.root.join('spec/support/schema.graphql'), 'w') do |f|
+      f.write(generated)
+    end
```

Comment:
The newline would make problems when comparing it to the current schema (see `ontohub_backend_schema_spec.rb`, though, wouldn't it?